test/Microsoft.AspNetCore.SignalR.Tests/TestClient.cs

```diff
@@ -28,7 +28,7 @@ public class TestClient : IDisposable, IInvocationBinder
         public Channel<byte[]> Application { get; }
         public Task Connected => ((TaskCompletionSource<bool>)Connection.Metadata["ConnectedTask"]).Task;
 
-        public TestClient(bool synchronousCallbacks = false)
+        public TestClient(bool synchronousCallbacks = false, bool addClaimId = false)
```

Comment:
I was just thinking - is there any reason to not add the claimId always (i.e. can we even not pass this parameter)?